backend/src/main/java/com/woowacourse/friendogly/club/domain/Club.java

```diff
@@ -0,0 +1,97 @@
+package com.woowacourse.friendogly.club.domain;
+
+import com.woowacourse.friendogly.member.domain.Member;
+import com.woowacourse.friendogly.pet.domain.Pet;
+import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Club {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Embedded
+    private Title title;
+
+    @Embedded
+    private Content content;
+
+    @Column(nullable = false)
+    private MemberCapacity memberCapacity;
+
+    @OneToOne(optional = false)
+    private Member owner;
+
+    @OneToMany
+    private List<Member> participantMembers = new ArrayList<>();
+
+    @OneToMany
+    private List<Pet> participantPets = new ArrayList<>();
+
+    //TODO: 필터링 조건
+
+    @Column(nullable = false)
+    private String imageUrl;
+
+    @Column(nullable = false)
+    @Temporal(TemporalType.TIMESTAMP)
+    private LocalDateTime createdAt;
```

Comment:
```suggestion
    @Column(nullable = false)
    private LocalDateTime createdAt;
```

`LocalDateTime`을 사용하면 생략 가능하다고 하네요!

Reply:
저희 배포 환경이 Mysql이라 , TIMESTAMP 타입으로 저장하려고 저렇게 명시해주긴 했어요.
제가 알기론 생략하면 Mysql에서  DATETIME으로 저장하는걸로 알고 있어서요
근데 좀 찾아보니깐 LocalDateTIme은 @Colume으로 지정해줘야된다고하네여 수정해야될 것 같아요

Reply:
좀 더 고민해보니깐 지금 단계에서 DATETIME으로 저장하나 타임스템프쓰나 별 차이 없는 것 같아서 지우도록 하겠습니당